src/plugins/tcl/README.md

```diff
@@ -2,7 +2,7 @@
 - infos/author = Markus Raab <elektra@libelektra.org>
 - infos/licence = BSD
 - infos/provides = storage/tcl
-- infos/needs = code null binary
+- infos/needs = ccode null base64
```

Comment:
as already said, it would be better if we can use generic providers here (like binary) and fix the tutorials to use concrete plugins (that pass ASAN checks)

Reply:
Would it make sense to solve this in a different PR?

Reply:
> we can use generic providers here (like binary) and fix the tutorials to use concrete plugins

I do not think that will work here, since these plugins are listed under `infos/needs` and not under `infos/recommends`.

Reply:
If you explicitly state needed plugins, kdb mount will know that the dependency is already fulfilled.